.envs/.local/.minio

```diff
@@ -0,0 +1,2 @@
+MINIO_ACCESS_KEY=enrolments
+MINIO_SECRET_KEY=wJalrXUtnFEMIK7MDENGbPxRfiCYEXAMPLEKEY
```

Comment:
I think we don't need them anymore. Did you check this PR: https://github.com/ACWIC/employer-admin/pull/27